R/pkg/R/DataFrame.R

```diff
@@ -510,9 +510,7 @@ setMethod("registerTempTable",
 #'
 #' Insert the contents of a SparkDataFrame into a table registered in the current SparkSession.
 #'
-#' @param x A SparkDataFrame
-#' @param tableName A character vector containing the name of the table
```

Comment:
I agree. that is an unfortunate side-effect. I wonder if there is a way to order this in the Rd generated roxygen; maybe this could be a reasonable PR to change roxygen for special casing `...`.
The only other solution I've seen would be to hand craft a .Rd file...

For now I think we should prioritize maintainability and keep the doc close to the function as much as possible.